### DISCUSSION TITLE
Add tags to multiple dashboards at once, from the Manage screen

### DISCUSSION THREAD

**Discussion by mjseaman** [grafana/grafana#38123] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
I'd like to be able to select multiple dashboards and apply a tag to them all at once.

**Why is this needed**:
I created a set of dashboards and decided to link between them using Dashboard links. My dashboard link relies on tags to filter for a specific set of dashboards. I'd like to multi-select some dashboards using checkboxes, and apply a tag to all of them at once. Otherwise, I need to click into the settings of each dashboard and adjust its tags individually.

It would also be nice to remove tags in the same way.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
